test/xapi/docs_test.rb

```diff
@@ -0,0 +1,49 @@
+require './test/test_helper'
+require 'xapi/docs'
+
+class DocsTest < Minitest::Test
+  def language
+    "fake"
+  end
+
+  def dir
+    './test/fixtures/'
+  end
+
+  def test_files
+    docs = Xapi::Docs.new(language, dir)
+    files = {
+      "HELLO.md" => "## Creating your first module\n",
```

Comment:
The contents of the fixtures can be even more obviously fake, e.g. just `hello`, `resources`, etc without worrying about markdown.